src/20240521_AaveV3Ethereum_OnboardingETHxToAaveV3/config.ts

```diff
@@ -0,0 +1,54 @@
+import {ConfigFile} from '../../generator/types';
+export const config: ConfigFile = {
+  rootOptions: {
+    pools: ['AaveV3Ethereum'],
+    title: 'Onboarding ETHx to Aave V3',
+    shortName: 'OnboardingETHxToAaveV3',
+    date: '20240521',
+    author: 'karpatkey_TokenLogic',
+    discussion: 'https://governance.aave.com/t/arfc-onboarding-ethx-to-aave-v3-ethereum/15672',
+    snapshot:
+      'https://snapshot.org/#/aave.eth/proposal/0x9238b091250c739f5b5486ab8dbaa110b0b7ec0582698ea2c2d3721377e4b0bb',
+    votingNetwork: 'POLYGON',
+  },
+  poolOptions: {
+    AaveV3Ethereum: {
+      configs: {
+        ASSET_LISTING: [
+          {
+            assetSymbol: 'ETHx',
+            decimals: 18,
+            priceFeed: '0xC5f8c4aB091Be1A899214c0C3636ca33DcA0C547',
```

Comment:
```suggestion
            priceFeed: '0xD6270dAabFe4862306190298C2B48fed9e15C847',
```
changing to capo adapter address instead of the ratio feed address